model/pdata/common.go

```diff
@@ -760,7 +760,7 @@ func (sm StringMap) Len() int {
 //
 // Example:
 //
-//   it := sm.Range(func(k string, v StringValue) {
+//   sm.Range(func(k string, v StringValue) {
```

Comment:
func returns `bool`

Reply:
Yes, func returns `bool` but `Range` returns nothing. This was assigning something to the output of `Range`.